pkg/rules/rulespb/rpc.proto

```diff
@@ -119,13 +119,14 @@ message Alert {
     string name                               = 2 [(gogoproto.jsontag) = "name" ];
     string query                              = 3 [(gogoproto.jsontag) = "query" ];
     double duration_seconds                   = 4 [(gogoproto.jsontag) = "duration" ];
-    ZLabelSet labels                          = 5 [(gogoproto.jsontag) = "labels", (gogoproto.nullable) = false ];
-    ZLabelSet annotations                     = 6 [(gogoproto.jsontag) = "annotations", (gogoproto.nullable) = false ];
-    repeated AlertInstance alerts             = 7 [(gogoproto.jsontag) = "alerts" ];
-    string health                             = 8 [(gogoproto.jsontag) = "health" ];
-    string last_error                         = 9 [(gogoproto.jsontag) = "lastError,omitempty" ];
-    double evaluation_duration_seconds        = 10 [(gogoproto.jsontag) = "evaluationTime" ];
-    google.protobuf.Timestamp last_evaluation = 11 [(gogoproto.jsontag) = "lastEvaluation", (gogoproto.stdtime) = true, (gogoproto.nullable) = false ];
+    double keep_firing_for_seconds            = 5 [(gogoproto.jsontag) = "keepFiringFor" ];
```

Comment:
I never did code anything with protocol buffer so I don't know if it's frowned upon to add a new attribute in the middle of an already existing structure.
I can change this if it causes any issue.

Reply:
Got my anser, it broke `TestQueryCompatibilityWithPreInfoAPI` e2e test!